### ISSUE TITLE
Generating Scribble annotations

### ISSUE THREAD

**Issue by GivralNguyen** [MIC-DKFZ/nnUNet#2124]
Hi, i recently read into your new work "Embarrassingly Simple Scribble Supervision for
3D Medical Segmentation". I saw that partial loss is already implemented in nnUnet (https://github.com/MIC-DKFZ/nnUNet/blob/master/documentation/ignore_label.md). Do you also have the code for the scribble generation from dense reference segmentations?
Thanks for your great work!

**Comment by FabianIsensee** [comment]
Hey, we are planning to release this code upon acceptance of the paper. I am also tagging @Karol-G here so that he is aware of this issue

**Comment by Karol-G** [comment]
Most likely yes, but of course only the final test set performance can say that :)
Do not forget to also exchange the background in the sparsely labeled data with the ignore label and to additionally provide some sparse labels for the background (in case that is not already the case).

**Comment by FabianIsensee** [comment]
From a technical perspective this should work. From a practical perspective one could think about how training data is sampled. Currently we sample cases randomly, regardless of how densely they are labeled. This means that scribbled cases will have the same importance in training as densely labeled cases. Whether that is a good thing or not I don't know

**Comment by GivralNguyen** [comment]
We will test it out and let you guys know how it goes

**Comment by FabianIsensee** [comment]
How did it go?
